proto/src/determined/api/v1/api.proto

```diff
@@ -180,7 +202,7 @@ service Determined {
   rpc PatchUser(PatchUserRequest) returns (PatchUserResponse) {
     option (google.api.http) = {
       patch: "/api/v1/users/{user_id}",
-      body: "user"
+      body: "*"
```

Comment:
why did this get changed?

Reply:
Had to change this so that I could also send isHashed (which is a part of the PatchUserRequest) instead of just the user field in PatchUserRequest.

Reply:
so the problem is that this is breaking the API payload format for the requests, because instead of patching `user` (i.e. `PatchUser` message) it becomes a nested structure.
this is also a problem because a EE customer is already using this as a raw REST API.
I think we should keep `body: "user"` here, but put `isHidden` into `PatchUser` object instead. I believe that should achieve the same functionality but non-breaking.

Reply:
Ok, I made this change in react too I will revert that.